Dockerfile.dapper

```diff
@@ -21,69 +21,35 @@ ENV GOPATH=/go PATH=/go/bin:/usr/local/go/bin:${PATH} SHELL=/bin/bash
 ENV CATTLE_KDM_BRANCH=release-v2.7
 
 RUN yum-config-manager --enable ol7_optional_latest && \
-    yum-config-manager --enable ol7_addons
-
-RUN yum update -y && \
-    # software collections repo needed for git 2.x on OL7
+    yum-config-manager --enable ol7_addons && \
     yum-config-manager --add-repo=http://yum.oracle.com/repo/OracleLinux/OL7/SoftwareCollections/x86_64 && \
-    yum install -y bash rh-git227 gcc glibc-static docker-cli vim less libffi file curl wget ca-certificates unzip tar jq \
-                   openssl-devel libffi-devel python3 iproute oracle-golang-release-el7 && \
+    yum-config-manager --add-repo https://yum.oracle.com/repo/OracleLinux/OL7/olcne13/x86_64/ && \
+    yum-config-manager --add-repo https://yum.oracle.com/repo/OracleLinux/OL7/olcne15/x86_64/ && \
     yum-config-manager --enable ol7_developer_golang119 && \
-    yum install -y golang-1.19 && \
+    yum install oracle-golang-release-el7 -y
```

Comment:
I have found that adding a "yum clean all" will reduce the size of a layer that you do a yum install in

Reply:
--squash will similarly reduce layer size